### DISCUSSION TITLE
Custom Colors for ProgressSpinner

### DISCUSSION THREAD

**Discussion by msg-tbrtz** [primefaces/community#1239] [PrimeNG]
Currently, the [ProgressSpinner ](https://primeng.org/progressspinner) component changes it's color through all colors of the rainbow, independent from the theme. It would be desirable to configure the color, e.g. to a single color.
For our case, we have a mostly bi-color theme; the colorful spinner disturbs that design.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Reply by msg-ptavoul** [reply]
It's not a bug, it's a new feature request.
We want a new API implementation with new attributes like _"spinnerColor": "red/green/blue..."_

Is it possible?

**Reply by cetincakiroglu** [reply]
Thanks for the feedback, we'll wait for community upvotes for this one.

**Reply by zbrown001** [reply]
I would like this feature as well!

**Comment by cagataycivici** [comment]
```js
const MyPreset = definePreset(Aura, {
    components: {
        progressspinner: {
            colorScheme: {
                light: {
                    root: {
                        colorOne: 'blue',
                        colorTwo: 'blue',
                        colorThree: 'blue',
                        colorFour: 'blue'
                    }
                },
                dark: {
                    root: {
                        colorOne: 'blue',
                        colorTwo: 'blue',
                        colorThree: 'blue',
                        colorFour: 'blue'
                    }
                }
            }
        }
    }
});
```
